### PR TITLE
[PSR4] Fix test on NormalizeNamespaceByPSR4ComposerAutoloadRector

### PR DESCRIPTION
**Pull request by samsonasik**

### CODE REVIEW

**Comment by samsonasik** [comment]
it works 🎉 🎉 🎉 All checks have passed 🎉 @TomasVotruba it is ready for review.

File: rules/PSR4/Rector/FileWithoutNamespace/NormalizeNamespaceByPSR4ComposerAutoloadRector.php [hunk 1]
```diff
@@ -82,33 +82,37 @@ public function getNodeTypes(): array
      */
     public function refactor(Node $node): ?Node
     {
-        if ($this->inlineHTMLAnalyzer->hasInlineHTML($node)) {
+        $processNode = clone $node;
+        if ($this->inlineHTMLAnalyzer->hasInlineHTML($processNode)) {
             return null;
         }
```
Comment (TomasVotruba) [inline rules/PSR4/Rector/FileWithoutNamespace/NormalizeNamespaceByPSR4ComposerAutoloadRector.php:88]: Why is the node cloned here?
Some test case would be helpful to explain what is wrong and what was fixed.

Comment (samsonasik) [reply rules/PSR4/Rector/FileWithoutNamespace/NormalizeNamespaceByPSR4ComposerAutoloadRector.php:88]: without this, there is error like in latest merged PR https://github.com/rectorphp/rector-src/runs/4000894749?check_suite_focus=true

do you have idea how to verify that?

Comment (samsonasik) [reply rules/PSR4/Rector/FileWithoutNamespace/NormalizeNamespaceByPSR4ComposerAutoloadRector.php:88]: @TomasVotruba the test error is came from previous merged PR which solved by this patch

https://github.com/rectorphp/rector-src/runs/4000894749?check_suite_focus=true

Comment (TomasVotruba) [reply rules/PSR4/Rector/FileWithoutNamespace/NormalizeNamespaceByPSR4ComposerAutoloadRector.php:88]: I'll look into it...

Comment (TomasVotruba) [reply rules/PSR4/Rector/FileWithoutNamespace/NormalizeNamespaceByPSR4ComposerAutoloadRector.php:88]: I'm looking into it and I cannot find anything. 
Looks good :+1:

**Pull request merged**